src/tokens/selectors.ts

```diff
@@ -495,3 +493,11 @@ export const feeCurrenciesWithPositiveBalancesSelector = createSelector(
 
 export const visualizeNFTsEnabledInHomeAssetsPageSelector = (state: RootState) =>
   state.app.visualizeNFTsEnabledInHomeAssetsPage
+
+export const importedTokensSelector = createSelector([tokensListSelector], (tokenList) => {
+  if (!getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW)) {
```

Comment:
great :D i think this is fine, since the value should be updated once per app session. another way we can do it without this assumption is to create a dummy selector for the feature flag to handle if the value does update (e.g. https://github.com/valora-inc/wallet/blob/fix-imported-tokens-balances/src/positions/selectors.ts#L17)